examples/sampling/pyg/node_classification.py

```diff
@@ -0,0 +1,268 @@
+    #   the input, hidden, and output features, respectively.
+    # - The forward method defines the computation performed at every call.
+    #####################################################################
+    def __init__(self, in_size, hidden_size, out_size):
+        super(GraphSAGE, self).__init__()
+        self.layers = torch.nn.ModuleList()
+        self.layers.append(SAGEConv(in_size, hidden_size))
+        self.layers.append(SAGEConv(hidden_size, hidden_size))
+        self.layers.append(SAGEConv(hidden_size, out_size))
+
+    def forward(self, blocks, x, device):
+        h = x
+        for i, (layer, block) in enumerate(zip(self.layers, blocks)):
+            src, dst = block.edges()
+            edge_index = torch.stack([src, dst], dim=0)
+            h_src, h_dst = h, h[: block.number_of_dst_nodes()]
+            h = layer((h_src, h_dst), edge_index)
+            if i != len(blocks) - 1:
+                h = F.relu(h)
+        return h
+
+
+def create_dataloader(dataset_set, graph, feature, device, is_train):
+    #####################################################################
+    # (HIGHLIGHT) Create a data loader for efficiently loading graph data.
+    #
+    # - 'ItemSampler' samples mini-batches of node IDs from the dataset.
+    # - 'sample_neighbor' performs neighbor sampling on the graph.
+    # - 'FeatureFetcher' fetches node features based on the sampled subgraph.
+    # - 'CopyTo' copies the fetched data to the specified device.
+
+    #####################################################################
+    # Create a datapipe for mini-batch sampling with a specific neighbor fanout.
+    # Here, [10, 10, 10] specifies the number of neighbors sampled for each node at each layer.
+    # We're using `sample_neighbor` for consistency with DGL's sampling API.
+    # Note: GraphBolt offers additional sampling methods, such as `sample_layer_neighbor`,
+    # which could provide further optimization and efficiency for GNN training.
+    # Users are encouraged to explore these advanced features for potentially improved performance.
+
+    # Initialize an ItemSampler to sample mini-batches from the dataset.
+    datapipe = gb.ItemSampler(
+        dataset_set, batch_size=1024, shuffle=is_train, drop_last=is_train
+    )
+    # Sample neighbors for each node in the mini-batch.
+    datapipe = datapipe.sample_neighbor(graph, [10, 10, 10])
+    # Fetch node features for the sampled subgraph.
+    datapipe = datapipe.fetch_feature(feature, node_feature_keys=["feat"])
+    # Copy the data to the specified device.
+    datapipe = datapipe.copy_to(device=device)
+    # Create and return a DataLoader to handle data loading.
+    dataloader = gb.DataLoader(datapipe, num_workers=0)
+
+    return dataloader
+
+
+def compute_accuracy(logits, labels):
+    _, predicted = torch.max(logits, dim=1)
+    correct = (predicted == labels).sum().item()
+    return correct / labels.size(0)
```

Comment:
```suggestion
```
Can we not use `MF.accuracy` everywhere?

Reply:
OK，where shall we not us MF.accuracy

Reply:
Sorry, I meant to use MF.accuracy everywhere.